.github/workflows/e2e_test.yml

```diff
@@ -133,15 +141,19 @@ jobs:
           minikube image load ./output/saved-images/chaos-mesh.tgz
           minikube image load ./output/saved-images/e2e-helper.tgz
       - name: Setup Helm
-        uses: azure/setup-helm@v1
+        uses: azure/setup-helm@v2.1
+      - name: Setup Kubectl
+        uses: azure/setup-kubectl@v2.0
+        with:
+          version: ${{ matrix.kubernetes-version }}
       - name: Install Chaos Mesh
         run: |
           # if kubernetes does not support apiextensions.k8s.io/v1, fallback to apiextensions.k8s.io/v1beta1
           if kubectl api-resources | grep -w apiextensions.k8s.io/v1; then
-            helm install --wait --create-namespace chaos-mesh helm/chaos-mesh --namespace=chaos-testing --set dnsServer.create=true --set dashboard.create=true --set images.tag=latest
+            helm install --wait --create-namespace chaos-mesh helm/chaos-mesh --namespace=chaos-testing --set dnsServer.create=${{ matrix.arch == 'x86_64' }} --set dashboard.create=true --set images.tag=latest
           else
             kubectl create --validate=false -f ./manifests/crd-v1beta1.yaml && \
-            helm install --skip-crds --wait --create-namespace chaos-mesh helm/chaos-mesh --namespace=chaos-testing --set dnsServer.create=true --set dashboard.create=true --set images.tag=latest
+            helm install --skip-crds --wait --create-namespace chaos-mesh helm/chaos-mesh --namespace=chaos-testing --set dnsServer.create=${{ matrix.arch == 'x86_64' }} --set dashboard.create=true --set images.tag=latest
```

Comment:
There are several Chaos that could not work with arm64. 

We should also update the matrix with `exclude`